README.md

```diff
@@ -88,35 +88,20 @@ sudo apt-get update
 sudo apt-get install git build-essential binutils-mips-linux-gnu python3
 ```
 
-#### 2. Download and set up qemu-irix (optional)
-
-Note: We are currently testing a recompiled version of the compiler that does not require qemu-irix.
-This step is optional and allows you to build with qemu-irix and the original compiler by adding `ORIG_COMPILER=1` to the `make` command in step 6, for example if you experience issues with the recompiled version.
-
-Download qemu-irix from the Releases section in the repository. Place it at a location of your choosing.
-
-Open up your .bashrc file (~/.bashrc), scroll to the bottom, and add the following, replacing the paths as necessary:
-
-```text
-export QEMU_IRIX="path/to/your/qemu-irix"
-```
-
-Save and close/reopen your terminal window.
-
-#### 3. Clone the repository
```

Comment:
What about moving this to a file in `docs/` with instruction on how to set up this instead of removing it completely?

Reply:
imo, no need. We kept the old ido method around incase recomp had any issues, but we've been using it for months with no hiccups at all. This method is worse in every way and really should never be needed going forward.
We'll still have the files and now how to use it in the very rare case we do need to use it. But the chance of that happening is really low

Reply:
I think at the level that QEMU is actually needed (no known cases) this is best stored in Discord.

Reply:
I remember one guy a few months ago needed to use the QEMU solution to properly setup decomp on their machine, so there's 1 known case.

Reply:
I think having a pin with these instructions in Discord should be enough. This way we at least have somewhere we can point to in case the setup is needed.

Reply:
Okey.
(Please resolve this conversation)